### ISSUE TITLE
Feature request - info and link for current photo

### ISSUE THREAD

**Issue by xss** [thesamet/webilder#14]
As there is an option to delete current photo, I'd really love to have an option to show info on the photo currently on the desktop, like right-clicking the Webilder icon and selecting "Properties of Current" or something, which triggers the same "Photo Properties" window as when browsing the collection and right-clicking on any photo and selecting "Properties...".

Very often I have an interesting photo on my background that I would love to be able to show to someone (i.e. via the flickr link), or to save to my personal collection. But searching that exact photo among the hundreds and thousands of photos inside the several (and monthly increasing number of) "Interestingness" albums, just to find out the photo's origin in its properties window -- that is just a pain in the behind, it can take ages! So far, I sometimes resort to taking a screenshot to save a photo, which is much faster and easier, but not a solution, especially as it also won't give credits to the original photographer.

One might think that the automatically created "Recent Photos" album would show recently displayed photos, but there I barely ever find any of the recent wallpapers. What does this "Recent Photos" display? Recently downloaded ones I assume, right?

Anyway, please make such an option to have the properties dialog for the current photo available. As you can already delete the current, I assume that shouldn't be too hard to implement, Webilder can already find and handle the current photo.

**Comment by borisdayma** [comment]
At least, it would be nice to go on the webilder icon/applet and have some details on the current wallpaper.

**Comment by mittfh** [comment]
Good idea - in my case, I've oodles of photos from Webshots, and it would be handy to be able to click an option to locate and highlight the currently displayed desktop wallpaper from within the many collections of photos I have.

**Comment by purdy** [comment]
I miss this feature. Before I upgraded Ubuntu 10.04 to 11.04, I just had to hover over the icon in the panel and a tooltip would show the current picture information. Now Ubuntu has changed with unity & whatever and I've lost that functionality.

**Comment by xss** [comment]
A while back, I had implemented this feature. Though in an early state, it was working fine for me, see issue #16.

I'm still planning to work on some improvements Nadav suggested – I just didn't find the time yet.. So, if anyone would like to continue where I left of to accelerate things, feel free to create your own fork and send Nadav a pull request when done. I really don't know when will be able to work on it again... Sorry!

EDIT: Oh, and as I am on Ubuntu 10.04, and therefore don't use Unity or Chrome Shell or whatever else Canonical is forcing on us these days, I assume there might be some issues and special testing necessary now with those new Ubuntu versions...

EDIT 2: If anyone indeed does decide to take up work on this feature, please let me know (and possibly also Nadav), so we won't do double work. And I could also provide you with the latest minor changes I made (removed comments, tidied up the new code a bit, see Nadav's suggestions) some time ago. :-)
